src/main/java/com/birbit/jsonapi/JsonApiMetaDeserializer.java

```diff
@@ -36,6 +38,14 @@ public JsonApiMeta deserialize(JsonElement json, Type typeOfT, JsonDeserializati
             JsonElement value = entry.getValue();
             if (value.isJsonPrimitive()) {
                 result.put(entry.getKey(), entry.getValue().getAsString());
+            } else if (value.isJsonArray()) {
+                List<String> allValues = new ArrayList<>();
+                for (JsonElement element : entry.getValue().getAsJsonArray()) {
+                    if (element.isJsonPrimitive()) {
+                        allValues.add(element.getAsJsonPrimitive().getAsString());
+                    }
+                }
+                result.put(entry.getKey(), allValues);
```

Comment:
JSON API spec:

Where specified, a meta member can be used to include non-standard meta-information. The value of each meta member MUST be an object (a “meta object”).

```
{
  "meta": {
    "copyright": "Copyright 2015 Example Corp.",
    "authors": [
      "Yehuda Katz",
      "Steve Klabnik",
      "Dan Gebhardt",
      "Tyler Kellen"
    ]
  },
  "data": {
    // ...
  }
}
```

Added support to parse arrays.

Reply:
Interesting that the API is using the meta blob for that 👍